src/FieldDefinition.hack

```diff
@@ -70,4 +69,41 @@ final class FieldDefinition<TParent, TRet, TResolved> implements IResolvableFiel
     public function getArguments(): dict<string, ArgumentDefinition> {
         return $this->arguments;
     }
+
+    public function getDeprecationReason(): ?string {
+        // TODO
+        return null;
+    }
+
+    public function isDeprecated(): bool {
+        // TODO
+        return false;
+    }
+
+    public function getDescription(): ?string {
+        // TODO
+        return null;
+    }
+
+    public function getArgs(): vec<Introspection\__InputValue> {
+        return Vec\map(
+            $this->arguments,
+            $arg ==> {
+                $input_value = shape(
+                    'name' => $arg['name'],
+                    'type' => $arg['type'],
+                );
+
+                if (Shapes::keyExists($arg, 'description')) {
+                    $input_value['description'] = $arg['description'];
+                }
+
+                if (Shapes::keyExists($arg, 'default_value')) {
+                    $input_value['defaultValue'] = (string)$arg['default_value'];
```

Comment:
This string needs to use GraphQL syntax (what you would write in a GraphQL query), e.g. string literals are quoted `"foo"`, lists are `[1, 2, 3]`, etc. I'm not sure if we have any default values where this matters, so it can possibly be addressed later.

A nice way to implement this would be to add a new method like `valueToString` to `IInputType`, then you can do `$arg['type']->valueToString($arg['default_value'])` here (or do that in the codegen and put the returned string in the generated files)

Reply:
Ah good call. Going to do this in a separate PR: https://github.com/mwildehahn/hack-graphql/issues/103